### PR TITLE
Remove useless data-test-id

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/cheminfo/nmrium/issues/1697

### CODE REVIEW

File: test-e2e/core/import.test.ts [hunk 1]
```diff
@@ -46,6 +46,6 @@ test('should load and migrate .nmrium data from version 2 to version 3', async (
   await nmrium.clickPanel('Filters');
 
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=Apodization'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Apodization'),
```
Comment (targos) [inline test-e2e/core/import.test.ts:49]: Does it work with `_react=FilterPanel` ? We should avoid "inner" or "internal" things if possible.

File: test-e2e/panels/filters.test.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ async function apodizationFilter(nmrium: NmriumPage) {
   await nmrium.page.click('button >> text=Apply');
 
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=Apodization'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Apodization'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:15]: ditto

File: test-e2e/panels/filters.test.ts [hunk 3]
```diff
@@ -21,29 +21,29 @@ async function zeroFillingFilter(nmrium: NmriumPage) {
   await nmrium.page.click('button >> text=Apply');
 
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=Zero Filling'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Zero Filling'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:24]: ditto

File: test-e2e/panels/filters.test.ts [hunk 4]
```diff
@@ -21,29 +21,29 @@ async function zeroFillingFilter(nmrium: NmriumPage) {
...
   ).toBeVisible();
 }
 
 async function fourierTransformFilter(nmrium: NmriumPage) {
   await nmrium.clickTool('fft');
 
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=FFT'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=FFT'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:32]: ditto

File: test-e2e/panels/filters.test.ts [hunk 5]
```diff
@@ -21,29 +21,29 @@ async function zeroFillingFilter(nmrium: NmriumPage) {
...
+  await nmrium.page.fill('_react=input[name="ph1"]', '-100');
+  await nmrium.page.fill('_react=input[name="ph0"]', '-104');
 
   // input debounce for 250ms
   await nmrium.page.waitForTimeout(250);
 
   await nmrium.page.click('button >> text=Apply');
   await expect(
-    nmrium.page.locator('data-test-id=filters-table >> text=Phase correction'),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Phase correction'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:46]: ditto

File: test-e2e/panels/filters.test.ts [hunk 6]
```diff
@@ -52,9 +52,7 @@ async function baselineCorrectionFilter(nmrium: NmriumPage) {
   await nmrium.page.click('button >> text=Apply');
 
   await expect(
-    nmrium.page.locator(
-      'data-test-id=filters-table >> text=Baseline correction',
-    ),
+    nmrium.page.locator('_react=FilterPanelInner >> text=Baseline correction'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:55]: ditto

File: test-e2e/panels/filters.test.ts [hunk 7]
```diff
@@ -83,7 +81,7 @@ test('process 1d FID 13c spectrum', async ({ page }) => {
   });
   await test.step('Check filters panel', async () => {
     await expect(
-      nmrium.page.locator('data-test-id=filters-table >> .filter-row'),
+      nmrium.page.locator('_react=FilterPanelInner >> .filter-row'),
```
Comment (targos) [inline test-e2e/panels/filters.test.ts:84]: ditto

File: test-e2e/core/import.test.ts (hunk 1 above)
Comment (wadjih-bencheikh18) [reply test-e2e/core/import.test.ts:49]: it works thanks

**Pull request merged**